opm/simulators/flow/FlowBaseVanguard.hpp

```diff
@@ -292,6 +329,19 @@ class FlowBaseVanguard : public BaseVanguard<TypeTag>,
              "See https://sandialabs.github.io/Zoltan/ug_html/ug.html "
              "for available Zoltan options.");
         Parameters::hideParam<TypeTag, Properties::ZoltanParams>();
+        Parameters::registerParam<TypeTag, Properties::MetisImbalanceTol>
+#if defined(IS_SCOTCH_METIS_HEADER) && IS_SCOTCH_METIS_HEADER
+            ("Tolerable imbalance of the loadbalancing provided by Scotch-METIS (default: 0.1).");
+#else
+            ("Tolerable imbalance of the loadbalancing provided by Metis (default: 1.1).");
+#endif
```

Comment:
Would be cool, if we could go without #if #else here specify both defaults in one help string